package.json

```diff
@@ -9,10 +9,10 @@
   ],
   "scripts": {
     "compile": "hardhat compile",
-    "coverage": "env COVERAGE=true hardhat coverage",
+    "coverage": "cross-env COVERAGE=true hardhat coverage",
```

Comment:
is env not working correctly ? I would not want to add a build dependency unless its absolutely necessary

Reply:
When I run this script, it was giving me an error and didn't work on my terminal. So, I added cross-env instead of env and it worked.